Source/ProjectFirma.Web/Views/MapLayer/EditExternalMapLayerViewModel.cs

```diff
@@ -0,0 +1,129 @@
+﻿/*-----------------------------------------------------------------------
+<copyright file="EditExternalMapLayerViewModel.cs" company="Tahoe Regional Planning Agency and Sitka Technology Group">
+Copyright (c) Tahoe Regional Planning Agency and Sitka Technology Group. All rights reserved.
+<author>Sitka Technology Group</author>
+</copyright>
+
+<license>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License <http://www.gnu.org/licenses/> for more details.
+
+Source code is available upon request via <support@sitkatech.com>.
+</license>
+-----------------------------------------------------------------------*/
+
+using LtInfo.Common.Models;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using LtInfo.Common;
+using ProjectFirma.Web.Common;
+using ProjectFirma.Web.Models;
+
+namespace ProjectFirma.Web.Views.MapLayer
+{
+    public class EditExternalMapLayerViewModel : FormViewModel, IValidatableObject
+    {
+        public int ExternalMapLayerID { get; set; }
+
+        [Required]
+        [StringLength(ExternalMapLayer.FieldLengths.DisplayName)]
+        [DisplayName("Display Name")]
+        public string DisplayName { get; set; }
+
+        [Required]
+        [StringLength(ExternalMapLayer.FieldLengths.LayerUrl)]
+        [DisplayName("Url")]
+        public string LayerUrl { get; set; }
+
+        [Required]

```

Comment:
Lets drop the Required on the booleans. Feels weird to me to see a required mark on a checkbox in a form.

Reply:
fixed